templates/title/show.json.twig

```diff
@@ -0,0 +1,47 @@
+{
+    "@context": "http://schema.org/",
+    "@type": "Book",
+    "@id": "{{ url('title_show', {'id':title.id}) }}",
+    "name": "{{title.title}}"
+    {% if title.pubdate %}, "datePublished": "{{title.pubdate}}" {% endif %}
+    {% if title.locationOfPrinting %},
+    "locationCreated": {
+        "@type": "Place",
+         "name": "{{title.locationOfPrinting}}" {% endif %},
+        "geo": {
+            "@type": "GeoCoordinates",
+            "latitude": "{{title.locationOfPrinting.latitude}}",
+            "longitude": "{{title.locationOfPrinting.longitude}}"
+        }
+    }
+    {#{% if title.imprint %}, "publisherImprint": "{{title.imprint}}" {% endif %}#},
+    "author": [
+            {% for titleRole in title.getTitleRoles("Author") %}{% set author = titleRole.person %}{
+            "@type": "Person"
+            {% if (author.firstName) and (author.lastName) %}, "name": "{{ author.firstName }} {{ author.lastName }}"  {% endif %},
+            "id": "{{ url('person_show', {'id':author.id}) }}"
```

Comment:
In some contexts it's @id and in some it's id (without the @). Can you explain the logic for choosing one or the other? This isn't something that needs to change, but I'd like to understand how and when it's @id.

Reply:
I believe it was only typos where I did not include the @, probably because it was close to the id of the actual entity (author.id)! But I corrected those.